packages/blob/src/multipart/complete.ts

```diff
@@ -72,6 +72,7 @@ export async function completeMultipartUpload({
           'x-mpu-key': encodeURI(key),
         },
         body: JSON.stringify(parts),
+        signal: options.abortSignal,
```

Comment:
Curious: Why is this `options.abortSignal` and in list this is `options?.abortSignal`?

Reply:
all put operations require an options object to set `access: "public"`